cmd/kubeadm/app/constants/constants.go

```diff
@@ -233,9 +233,9 @@ var (
 
 	// SupportedEtcdVersion lists officially supported etcd versions with corresponding kubernetes releases
 	SupportedEtcdVersion = map[uint8]string{
-		8:  "3.0.17",
 		9:  "3.1.10",
-		10: "3.1.10",
+		10: "3.2.12",
+		11: "3.2.12",
```

Comment:
yeah it's too early to add supported etcd version of 1.11. Add it here to make the test of "experimental version" pass. It may change in the future.

Reply:
looks good to me